src/components/MyBooksItem.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```

Comment:
Is it possible to reuse the BookItem component you recreated earlier ? This way we have lesser maintainability.